packages/components/psammead-episode-list/src/MediaIndicator.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string } from 'prop-types';
+
+const Wrapper = styled.div`
+  display: inline-block;
+  width: ${props => props.size};
+  height: ${props => props.size};
+`;
+
+const MediaIndicator = ({ size }) => (
+  <Wrapper
+    aria-hidden="true"
+    size={size}
+    dangerouslySetInnerHTML={{
+      __html: `
+      <svg class="rounded-play-button" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40">
+          <path class="rounded-play-button__outer-circle" d="M20,0C8.954,0,0,8.954,0,20s8.954,20,20,20s20-8.954,20-20S31.046,0,20,0z"/>
+          <path class="rounded-play-button__inner-circle" fill="#fff" d="M20,1.765C9.929,1.765,1.765,9.929,1.765,20S9.929,38.235,20,38.235S38.235,30.071,38.235,20
```

Comment:
You may need to remove the fill colour here and use CSS to specify the colour? The icon should be viewable without CSS and also change colour if a user changes their colour preferences in Firefox or uses a custom mode on Windows High contrast mode. See our other media icons and points 6 and 7 on here https://bbc.github.io/accessibility-news-and-you/accessibility-news-and-testers